.github/workflows/docker_build_push.yml

```diff
@@ -0,0 +1,77 @@
+name: docker-build-push
+
+on:
+  workflow_call:
+
+env:
+  DOCKERHUB_API: https://hub.docker.com/v2
+
+jobs:
+  build-push:
+    runs-on: [ self-hosted, Linux, x86_64, regular ]
+
+    steps:
+      - name: Prepare checkout
+        uses: tarantool/actions/prepare-checkout@master
+
+      - uses: actions/checkout@v4
+
+      - uses: ./.github/actions/environment
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Compose image tags
+        run: |
+          REPO=tarantool/tarantool
+          case ${{ github.ref }} in
+            refs/tags/*-alpha*|refs/tags/*-beta*|refs/tags/*-rc*)
+              echo "TAGS=${REPO}:${{ github.ref_name }}" >> $GITHUB_ENV
+              ;;
+            refs/tags/*)
+              MAJOR=$(echo ${{ github.ref_name }} | cut -f1 -d.)
+              MINOR=$(echo ${{ github.ref_name }} | cut -f2 -d.)
+              MAJOR_MINOR=${MAJOR}.${MINOR}
+              MAJOR_MINOR_UP=${MAJOR}.$((MINOR + 1))
```

Comment:
Why +1? At least, it requires some clarification. I don't understand.

Reply:
Clarification added.